src/GitHub/Data/Issues.hs

```diff
@@ -43,7 +43,7 @@ instance Binary Issue
 data NewIssue = NewIssue
     { newIssueTitle     :: !Text
     , newIssueBody      :: !(Maybe Text)
-    , newIssueAssignee  :: !(Maybe Text)
+    , newIssueAssignees :: !(Maybe (Vector (Name User)))
```

Comment:
is there difference between `null`/non-existing field (Nothing) and `[]` (empty Vector)?

Reply:
I believe sending `[]` causes all assignees to to be removed. See the notes for `assignees` on https://developer.github.com/v3/issues/#edit-an-issue

Reply:
... in `NewIssue` ?

Reply:
I've confirmed that both an empty vector and having the keys missing are accepted by GitHub, so really it's a design question if you would prefer to send `"assignees": []` or not.